### ISSUE TITLE
Univariate results

### ISSUE THREAD

**Issue by kevinykuo** [kasaai/pc-pricing-tutorial#91]
aka one-way analysis

**Comment by kevinykuo** [comment]
@marcopark90 you should be most of the way there -- wanna take this on?

**Comment by marcopark90** [comment]
@kevinykuo Yes, sure. I will create a function to make summary table and the relative graphs
